### PR TITLE
[ICP-13411] Qubino Flush Shutter - energy usage is not reported

### PR DESCRIPTION
**Pull request by PKacprowiczS**
@greens @SmartThingsCommunity/srpol-pe-team

### CODE REVIEW

File: devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy [hunk 1]
```diff
@@ -82,6 +82,7 @@ def installed() {
 	state.temperatureSensorDni = null
 	sendHubCommand(encap(zwave.configurationV2.configurationGet(parameterNumber: 71)))
 	sendEvent(name: "checkInterval", value: 2 * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	schedule(new Date(), getEnergyUsage)
```
Comment (greens) [inline devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy]: Any reason you're using `schedule` here and not `runIn` or just sending the hub command from within here?

Comment (PKacprowiczS) [reply devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy]: I want it to run every day, as I've described in the ticket and according to [the documentation](https://docs.smartthings.com/en/latest/smartapp-developers-guide/scheduling.html#run-on-a-recurring-schedule) using `schedule` with `new Date()` is one way to do it.

Comment (greens) [reply devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy]: huh, interesting

**Review by greens — DISMISSED** [review]
lgtm, but will wait to see if this is enough for cert team

**Comment by PKacprowiczS** [comment]
Sorry for the mess, accidentally pushed to a wrong branch

**Review by greens — DISMISSED** [review]

**Comment by PKacprowiczS** [comment]
@greens cert has verified it and it's ready to merge

**Review by greens — APPROVED** [review]

**Pull request merged**